basis/common-model/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>basis</artifactId>
+        <groupId>com.softgroup</groupId>
+        <version>0.0.1</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>common-model</artifactId>
```

Comment:
ok. I am going to create module "common-dao". In this module I want to create package "model" and put all models in it(Contact, Conversation, ConvDetails, ConvSettings, Cursor, Message, Profile, ProfSettings, ProfStatus). All dao (MessageDao and so on) i will put in "auth-api", "profile-api", "messanger-api". Will it be right?